packages/@aws-cdk/integ-runner/package.json

```
@@ -71,7 +71,7 @@
   },
   "dependencies": {
     "chokidar": "^3.6.0",
-    "@aws-cdk/cloud-assembly-schema": "0.0.0",
+    "@aws-cdk/cloud-assembly-schema": "^36.0.1",
     "@aws-cdk/cloudformation-diff": "0.0.0",
     "@aws-cdk/cx-api": "0.0.0",
```

Amazon Q: Please use the correct version as shown below
```suggestion
    "@aws-cdk/cx-api": "^37.0.1",
```

---

This version of cx-api doesn't exist.